assets/Js/script.js

```diff
@@ -0,0 +1,63 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const listContainer = document.getElementById('display-book-list');
+const addbookForm = document.getElementById('add-form');
+const bookStorage = [];
+
+// Add books and store in localstorage
+function addBooks() {
+  const storedData = JSON.parse(localStorage.getItem('StoreBook')) || [];
+  const Book = {
+    title: title.value,
+    author: author.value,
+  };
```

Comment:
- [ ] [optional] @team great work on your application, however, it would be nice to have unique id property for each book since it would make it easier to remove books with duplicate values

Reply:
Thank you for your feedback, Updated as per the suggestion.